github_utils.py

```diff
@@ -1003,3 +1003,28 @@ def get_pr_commits_reversed(pr):
         except IndexError:
             print("Index error: May be PR with no commits")
     return []
+
+
+def enable_github_loggin():
+    import logging
+
+    class MyHandler(logging.Handler):
+        level = 0
```

Comment:
@iarspider , I think you can set it to `logging.DEBUG` to old get the calls for debug messages